decoder/src/elf2table/mod.rs

```diff
@@ -12,8 +12,8 @@ use std::{
     path::{Path, PathBuf},
 };
 
-pub use crate::decoder::Table;
-use crate::decoder::{StringEntry, TableEntry, Tag};
+pub use crate::Table;
```

Comment:
This can be made private, there's no longer any reason to reexport it

```suggestion
use crate::Table;
```